platform/src/main/resources/patterns/README.md

```diff
@@ -0,0 +1,44 @@
+**Grok**
+
+**Grok Basics**
```

Comment:
We don't say what grok is. Perhaps this section should be entitled: "Specifying rules for parsing logs".
We can then say that we rely on the grok library for this purpose, and put the link to the grok documentation.